types/TileInput.ts

```diff
@@ -24,6 +24,16 @@ export interface TileInput<T> extends TileRequest {
    */
   radius?: number;
 
+  /**
+   * @description The tile extent. Here the grid dimensions will be 4096x4096
+   */
+  extent?: number;
+
+  /**
+   * @description The buffer size. Here we'll have a margin of 256 grid cells in addition to the tile extent.
```

Comment:
The buffer around the tile extent in the number of grid cells as specified by ST_AsMVT https://postgis.net/docs/ST_AsMVT.html. The default is 256.